### PR TITLE
Task/update iotagent node library

### PR DESCRIPTION
**Pull request by AlvaroVega**
Reopen branch.
Some improvements must be done in order to upgrade last iota node library

### CODE REVIEW

File: lib/mappings.js [hunk 1]
```diff
@@ -23,7 +23,11 @@
 'use strict';
```
Comment (fgalan) [inline lib/mappings.js]: CNR entry corresponding to this PR is missing?

Comment (AlvaroVega) [reply lib/mappings.js]: This branch has been reopened.
CNR already is saying:  Update iotagent node library to allow use env variables

Comment (fgalan) [reply lib/mappings.js]: Are the new changes also related with env vars? If the scope of the adaptation work is wider maybe the message should be adjusted.

Comment (AlvaroVega) [reply lib/mappings.js]: The changes are related with the update of node iotagent library, that was due to need of env vars.

Comment (fgalan) [reply lib/mappings.js]: NTC

**Pull request merged**